data/campaigns/Legend_of_Wesmere/scenarios/chapter4/14_Human_Alliance.cfg

```diff
@@ -586,6 +586,7 @@ Chapter Four"
             result=victory
             bonus=no
             save=no
+            carryover_percentage=40
```

Comment:
I really think the balance adjustments should be playtested to ensure it's at least passable.

Reply:
Shouldn't that sort of thing have been done when everything was converted to 40% in the first place?